### PR TITLE
[SPARKTA-472] [PLUGINS] Ingestion parser extract types from output fields

### PR DESCRIPTION
**Pull request by compae**
***given*** an instance of Sparta with one ingestion parser
***when*** you add outputs fields with the frontend
***then*** the data fields are casted with the type selected in the outputFields

### CODE REVIEW

**Comment by danielcsant** [comment]
+1

File: plugins/src/test/scala/com/stratio/sparta/plugin/test/parser/ingestion/IngestionParserTest.scala [hunk 1]
```diff
@@ -96,12 +101,15 @@ class IngestionParserTest extends WordSpecLike with Matchers with BeforeAndAfter
...
-      val schema = StructType(Seq(StructField(Input.RawDataKey, StringType)))
-
-      val ingestionParser = new IngestionParser(ParserOrder, InputField, OutputsFields, schema, ParserConfig)
+      val ingestionParser = new IngestionParser(ParserOrder, InputField, OutputsFields, validSchema, ParserConfig)
       val event = ingestionParser.parse(inputEvent, false)
+      val values = Seq("columnAValue", 1L)
 
       event should be eq Row.fromSeq(Seq(json) ++ event.toSeq)
+
+      event.size should be (3)
```
Comment (anistal) [inline plugins/src/test/scala/com/stratio/sparta/plugin/test/parser/ingestion/IngestionParserTest.scala]: Is it necessary to check the size when you are checking the full event after?

Comment (compae) [inline plugins/src/test/scala/com/stratio/sparta/plugin/test/parser/ingestion/IngestionParserTest.scala]: Yes

**Comment by anistal** [comment]
+1

**Pull request merged**